Makefile

```diff
@@ -75,11 +75,12 @@ update-all: update-app-doc update-impi-doc
 update: update-all ## Update buildable docs from xml and docs/ directories.
 
 build-app-selinux:
-	$(foreach app,$(APPS), \
-	    make -C app/$(app)/selinux NAME=rabezbx$(subst -,,$(app)) -f ${SELINUX_MAKEFILE} || :; \
-	)
-	$(foreach app,$(APPS), \
-	    [[ -d app/$(app)/selinux ]] && echo -n 'rabezbx$(subst -,,$(app)) ' >> rabe.lst; \
+# SELinux type enforcement files are named rabezbx<app>.te and will result
+# in a SELinux module name of rabezbx<app>
+	$(foreach teFile,$(wildcard app/*/selinux/rabezbx*.te), \
+		make -C $(dir $(teFile)) -f $(SELINUX_MAKEFILE) \
+			PREFIX=/usr NAME=$(notdir $(basename $(teFile))) && \
+			echo -n "$(notdir $(basename $(teFile))) " >> rabe.lst; \
```

Comment:
This line should probably have one tab less to align with the make call above.

Reply:
I've indented it as it belongs to the same (logical) line as `make [...] && [...]`, but if you think it makes it more readable, I can remove one tab...

Reply:
how about

```
   [...] \
&& [...]
```

as a basic pattern (with readability being king).

Reply:
Addressed with 6c201ba19ec592548018e60eb7d1e87964e172f6